src/caret_analyze/refactored_plot/__init__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Research Institute of Systems Planning, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .metrics_base import MetricsBase
```

Comment:
This package is a class that is not used outside of factored_plot, so it can be excluded from factored_plot/__init__.py.

```suggestion
```

Reply:
Fixed in: https://github.com/tier4/CARET_analyze/pull/170/commits/fd1bbff770db58943291d5118de69375d0028f5e